document-generator-api/src/main/java/uk/gov/companieshouse/document/generator/api/utility/ApiResponseMapper.java

```diff
@@ -14,11 +14,12 @@ public ResponseEntity map(ResponseObject responseObject) {
             case CREATED:
                 return ResponseEntity.status(HttpStatus.CREATED).body(responseObject.getData());
             case NO_DATA_RETRIEVED:
+            case NO_DOCUMENT_TYPE_FOUND:
```

Comment:
Should this map to `HttpStatus.NOT_FOUND` rather than `BAD_REQUEST`?

Reply:
In the event that the Uri does not match the regex pattern to return a DocumentType it has been confirmed to deal with these as a BAD_REQUEST, the same as when no data is retrieved.